docs/Concepts_and_Terminology.rst

```diff
@@ -80,7 +81,7 @@ specification parameter (see :ref:`writing_easyconfigs_easyblock_spec`).
 Since EasyBuild v2.0, an easyblock *must* be specified in case no matching easyblock is found based on the
 software name (cfr. :ref:`depr_ConfigureMake_fallback_eb1`).
 
-.. XXX - UPDATE BY VERSION
+.. XXX - UPDATE BY VERSION FIXME
 
 EasyBuild version 2.4.0 includes 154 software-specific easyblocks and 28 generic
```

Comment:
it should be, yes, or we should have a way to auto-update it; I won't do it here though, we should automate it, since it'll go stale again if we don't